custom/index.md

```diff
@@ -47,10 +62,13 @@ defaults:
   layout: 'default'
   # ...
 
-# information for your slides, can be a markdown string
-info: |
```

Comment:
Wondering why this has been removed?

Reply:
I just moved it at the top to group it with other project information.
But I changed the value to `false` because the sentence above state that this is presenting the default values.